### PR TITLE
3.0/fix/topic label

### PR DESCRIPTION
**Pull request by jonatanhanson**
removed faulty id from label

### CODE REVIEW

**Comment by helsingborgsstad** [comment]
Task linked: [CU-fz4p2g Label för customer feedback hänvisar till icke-existerande ID](https://app.clickup.com/t/fz4p2g)

File: templates/form.php [hunk 1]
```diff
@@ -61,9 +61,9 @@
                     <!-- Topic segment --> 
                     <div id="customer-feedback-topics" class="customer-feedback-topics feedback-answer-no u-margin__top--4" style="display:none">
 
-                        <label for="customer-feedback-comment-topic-<?php echo $num; ?>" class="c-typography c-typography__variant--h3">
+                        <legend class="c-typography c-typography__variant--h3">
```
Comment (sebastianthulin) [inline templates/form.php]: It seems like the specification that legends always should be combined with the <fieldset> tag.

Comment (jonatanhanson) [reply templates/form.php]: vilken tag? ;D

Comment (sebastianthulin) [reply templates/form.php]: Ouch. Github stripped the tag from the comment. I was talking about the fieldset tag.

**Pull request merged**